pkg/service/service.go

```diff
@@ -40,6 +41,9 @@ func NewService(config *config.Config) *http.Server {
 		// TODO: start tracing & inject span to context
 		ctx := context.Background()
 		s.elector = election.StartElection(config)
+		if s.elector == nil {
+			return nil, errors.New("elector is nil")
```

Comment:
add `log` here?

Reply:
> add `log` here?

Do not print duplicate logs, this error can be displayed on the console.